Gemfile

```diff
@@ -8,3 +8,6 @@ gem 'rake'
 gem 'whenever'
 gem 'zipruby'
 gem 'rubocop'
+
+
+gem 'simplecov', require: false
```

Comment:
[Here's](https://github.com/umts/umasstransit.rodeo/blob/master/Gemfile#L40) an example of how to do that.